packages/common/core-utils/api-extractor-esm.json

```diff
@@ -0,0 +1,5 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../common/build/build-common/api-extractor-base-esm.json",
+	"mainEntryPointFilePath": "<projectFolder>/lib/index.d.ts"
```

Comment:
Why does core-utils override the `d.mts` version of this?

Reply:
Because core-utils does not yet use tsc-multi. Those changes are in https://github.com/microsoft/FluidFramework/pull/18881